src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -1699,6 +1699,27 @@ export default class Aircraft {
     }
 
     /**
+     * Checks if the aircraft is inside the airspace of a specified airport
+     *
+     * @for AircraftInstanceModel
+     * @method isInsideAirspace
+     * @param  {airport} airport the airport whose airspace we are checking
+     * @return {Boolean}
+     * @private
+     */
+    isInsideAirspace(airport) {
```

Comment:
you don't need to do this, but in cases like this where you only need a few properties from a massive object you could destructure them:
```javascript
isInsideAirspace(airport) { }
// becomes
isInsideAirspace({ perimeter, ctr_radius, ctr_ceiling }) { }
```